optuna/visualization.py

```diff
@@ -29,6 +29,17 @@
     _available = False
 
 
+def is_available():
+    # type: () -> bool
+    """Returns whether visualization is available or not.
+
+    Returns:
+        ``True`` is visualization is available, ``False`` otherwise.
```

Comment:
typo?
```suggestion
        ``True`` if visualization is available, ``False`` otherwise.
```

Reply:
You're right, thank you. Applied the suggestion.